components/avatar/avatar.css

```diff
@@ -0,0 +1,44 @@
+.avatar {
+    margin: 1rem;
+    border-radius: 50%;
+    width: 50px;
+    height: 50px;
+    position: relative;
+    display: inline;
+}
+
+.figure {
+    border-radius: 50%;
+}
+
+.img {
+    border-radius: 50%;
+    height: 100%;
+    width: 100%;
+}
+
+.avatar.square {
+    border-radius: 2px;
+}
+
+.avatar-status {
+    border: 2px solid #fff;
+    background-color: #bcc3ce;
+    padding: 8%;
+    border-radius: 50%;
+    position: absolute;
+    bottom: 0em;
+    right: 0em;
+}
+
+.avatar-status.offline {
+    background-color: #bcc3ce;
+}
+
+.avatar-status.dnd {
+    background-color: #ff0000;
+}
+
+.avatar-status.online {
+    background-color: #32b643;
+}
```

Comment:
For consistency, why don't you store these background colors in a variable as well, as you have done in rest of the code.

Reply:
Hi, this was done in the respective branch.